src/main/groovy/life/qbic/portal/sampletracking/web/views/SampleImport.groovy

```diff
@@ -60,19 +63,20 @@ class SampleImport extends VerticalLayout {
         this.singleSampleAddButton.addClickListener({ event ->
             // Get value from user input in textField
             String sampleIdInput = this.additionalSampleId.getValue()
+            if (sampleIdInput.isEmpty()) {
+                StyledNotification emptyId = new StyledNotification("Empty ID" ,"Please enter a sample ID and try again.", Notification.Type.ASSISTIVE_NOTIFICATION)
+                emptyId.show(Page.getCurrent())
+                return
```

Comment:
empty return?

Reply:
Yes it does nothing if no ID is given.

Reply:
is the return statement necessary then?

Reply:
Yes because we do not want to continue trying to add an empty ID